### PR TITLE
docs: update Platform Settings page PEM-3979

### PR DESCRIPTION
**Pull request by addetz**
## Describe the Change

This patch updates the Platform Settings page:
- Renames "Pause Platform Upgrades".
- Introduces tabs for easier navigation.
- Updates the page style to match the docs style guide. 
- Breaks up content into a folder & 2 pages.
- Sets up redirect from old URL. 

## Review Changes

💻 [Preview URL](https://655686716f52f107de2a9e3f--docs-spectrocloud.netlify.app/clusters/cluster-management/platform-settings/)

🎫 [PEM-3979](https://spectrocloud.atlassian.net/browse/PEM-3979)

### CODE REVIEW

**Comment by karl-cardenas-coding** [comment]
@addetz we only use `feat` or `fix` when merging the release branch. Otherwise, a new repository git tag will be created. Let's make sure the squashed commit for the PR has the `docs:` prefix 😄

**Comment by addetz** [comment]
> @addetz we only use commit message prefix `feat` or `fix` when merging the release branch. Otherwise, a new repository git tag will be created. Let's make sure the squashed commit for the PR has the `docs:` prefix 😄

All done now. I'll remember this for the future as well.

File: docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+<TabItem value="projectScope" label="Pause Upgrades for all Clusters within Project Scope">
+
+* Tenant admin access
+
+* An existing cluster. Follow this [tutorial](../../clusters.md) to get started.
+
+</TabItem>
+
+<TabItem value="tenantScope" label="Pause Upgrades for all Clusters within Tenant Scope">
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md:36]:
```suggestion
```

Let's remove the tabs for the prereqs. They too similar, so we can wordsmith the permissions sentence.

File: docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+* Tenant admin access
+
+* An existing cluster. Follow this [tutorial](../../clusters.md) to get started.
+
+</TabItem>
+
+</Tabs>
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md:44]:
```suggestion
```

File: docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+* Tenant admin access
+
+* An existing cluster. Follow this [tutorial](../../clusters.md) to get started.
+
+</TabItem>
+
+<TabItem value="tenantScope" label="Pause Upgrades for all Clusters within Tenant Scope">
+
+* Tenant admin access
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md:38]:
```suggestion
* Cluster Admin permissions or Tenant Admin permissions when pausing upgrades for all clusters within tenant scope.
```

File: docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+</TabItem>
+
+</Tabs>
+
+## Enablement
+
+<Tabs groupId="pauseUpgrades">
+
+<TabItem value="singleCluster" label="Pause Upgrades for a Single Cluster">
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md:50]:
```suggestion
<TabItem value="singleCluster" label="Single Cluster">
```

File: docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+
+4.  Select **Settings** > **Cluster Settings**.
+
+5. Toggle the **Pause Agent Upgrades** button to pause cluster upgrades.
+
+6. A pop-up box will ask you to confirm the action. Click **OK**.
+
+</TabItem>
+
+<TabItem value="projectScope" label="Pause Upgrades for all Clusters within Project Scope">
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md:66]:
```suggestion
<TabItem value="projectScope" label="All Clusters - Project Scope">
```

File: docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+
+3. Select **Platform Settings**.
+
+4. Toggle the **Pause Agent Upgrades** button to pause upgrades for all clusters within the project scope.
+
+5. A pop-up box will ask you to confirm the action. Click **OK**.
+
+</TabItem>
+
+<TabItem value="tenantScope" label="Pause Upgrades for all Clusters within Tenant Scope">
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md:80]:
```suggestion
<TabItem value="tenantScope" label="All Clusters - Tenant Scope">
```

File: docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+
+</TabItem>
+
+</Tabs>
+
+## Validate
+
+<Tabs groupId="pauseUpgrades">
+
+<TabItem value="singleCluster" label="Pause Upgrades for a Single Cluster">
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md:100]:
```suggestion
<TabItem value="singleCluster" label="Single Cluster">
```

File: docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+
+3. Select your cluster.
+
+4. From the cluster details page, click **Settings** > **Cluster Settings**.
+
+5. The **Pause Agent Upgrades** toggle button is checked.
+
+</TabItem>
+
+<TabItem value="projectScope" label="Pause Upgrades for all Clusters within Project Scope">
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md:114]:
```suggestion
<TabItem value="projectScope" label="All Clusters - Project Scope">
```

File: docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+
+2. Navigate to the left **Main Menu** and click on **Project Settings**.
+
+3. Select **Platform Settings**.
+
+4. The **Pause Agent Upgrades** toggle button is checked.
+
+</TabItem>
+
+<TabItem value="tenantScope" label="Pause Upgrades for all Clusters within Tenant Scope">
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md:126]:
```suggestion
<TabItem value="tenantScope" label="All Clusters - Tenant Scope">
```

File: docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+* Tenant admin access
+
+* An existing cluster. Follow this [tutorial](../../clusters.md) to get started.
+
+</TabItem>
+
+</Tabs>
+
+## Enablement
+
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md:47]:
```suggestion
## Enablement

Use the tabs below to view the instructions for enabling the **Pause Agent Upgrades** feature for a single cluster, all clusters within the project scope, or all within the tenant scope.
```
By adding this text, we can shorten the tab titles 😄

File: docs/docs-content/clusters/cluster-management/platform-settings/cluster-auto-remediation.md [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+
+Palette provides **Cluster Auto Remediation** as a node reconciliation operation. When Cluster Auto Remediation is on, unhealthy nodes in all the Palette-provisioned clusters will automatically be replaced with new nodes. Turning off this feature will disable auto remediation. This feature can work under two scopes:
+
+* Tenant
+
+* Project
+
+## Prerequisites
+
+* Tenant admin access
```
[the remark this answers is not in the document]
Comment (addetz) [reply docs/docs-content/clusters/cluster-management/platform-settings/cluster-auto-remediation.md:18]: Ooooh! I asked Spectromate which kind of user had access to project settings and it said that tenant admin was required. I did not realise there was project admin access.

File: docs/docs-content/clusters/cluster-management/platform-settings/cluster-auto-remediation.md [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+4. Toggle the **Cluster Auto Remediation** button.
+
+5. A pop-up box will ask you to confirm the action. Click **OK**.
+
+</TabItem>
+
+</Tabs>
+
+:::info
+This does not apply to EKS, AKS or TKE clusters.
```
[the remark this answers is not in the document]
Comment (addetz) [reply docs/docs-content/clusters/cluster-management/platform-settings/cluster-auto-remediation.md]: I see Jun has confirmed this and have moved up in the intro. I wonder if this is a true info box though? I'm not sure how optional it is for the user to know so i'm wondering if we should use a `::caution` instead.

[the remark this answers is not in the document]
Comment (addetz) [reply docs/docs-content/clusters/cluster-management/platform-settings/cluster-auto-remediation.md]: I've used `::caution` instead.

**Pull request merged**